src/content/developers/docs/scaling/validium/index.md

```diff
@@ -7,30 +7,66 @@ incomplete: true
+
+## Volitions and validium
+
+Validiums offer many benefits, but come with trade-offs (most notably, data availability). But, as with many scaling solutions, validiums are suited to specific use-cases—which is why volitions were created. 
+
+Volitions combine a ZK-rollups and validium chain and allow users to switch between the two scaling solutions. With volitions, users can take advantage of validium's off-chain data availability for certain transactions, while retaining the freedom to switch to an on-chain data availability solution (ZK-rollup) if needed. This essentially gives users freedom to choose trade-offs as dictated by their unique circumstances. 
+
+A decentralized exchange (DEX) may prefer using a validium’s scalable and private infrastructure for high-value trades. It can also use a ZK-rollup for users who want a ZK-rollup's higher security guarantees and trustlessness. 
+
+## Pros and cons of validium {#pros-and-cons-of-validium}
 | Pros                                                                                                      | Cons                                                                                                                                     |
 | --------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
 | No withdrawal delay (no latency to on-chain/cross-chain tx); consequent greater capital efficiency.       | Limited support for general computation/smart contracts; specialized languages required.                                                 |
 | Not vulnerable to certain economic attacks faced by fraud-proof based systems in high-value applications. | High computational power required to generate ZK proofs; not cost effective for low throughput applications.                             |
-|                                                                                                           | Slower subjective finality time (10-30 min to generate a ZK proof) (but faster to full finality because there is no dispute time delay). |
-|                                                                                                           | Generating a proof requires off-chain data to be available at all times.                                                                 |
+| Reduces rollup fees for users by shrinking costs of publishing data on Ethereum.                          | Slower subjective finality time (10-30 min to generate a ZK proof) (but faster to full finality because there is no dispute time delay). |
+| Suitable for specific use-cases, like trading or blockchain gaming that prioritize transaction privacy and scalability. | Generating a proof requires off-chain data to be available at all times.                                                   |
```

Comment:
Does validium prioritize privacy?

Reply:
Sort of. As I mentioned earlier, StarkWare suggests validium is relatively more private for users than zk-rollups with on-chain data availability.

Reply:
> Does validium prioritize privacy?

We could change that point to read: "Suitable for specific use-cases...that require lower trust assumptions/trustlessness." I think that highlights the major difference between zk-rollups and validiums. LMK if that works.

Reply:
I can see how Validiums could improve privacy from the same perspective as your transactions on a centralized exchange are "private." If the data isn't published it would be more private to the general public, but also increases the trust assumptions if you aren't the one holding that data yourself.